ingest/defaults/config.yaml

```diff
@@ -4,70 +4,38 @@
 # Define optional config parameters with their default values here so that users
 # do not have to dig through the workflows to figure out the default values
 
-# Required to fetch from NCBI Datasets
-ncbi_taxon_id: "11082"
-
-# The list of NCBI Datasets fields to include from NCBI Datasets output
-# These need to be the "mnemonics" of the NCBI Datasets fields, see docs for full list of fields
-# https://www.ncbi.nlm.nih.gov/datasets/docs/v2/reference-docs/command-line/dataformat/tsv/dataformat_tsv_virus-genome/#fields
-# Note: the "accession" field MUST be provided to match with the sequences
-ncbi_datasets_fields:
-  - accession
-  - sourcedb
-  - isolate-lineage
-  - geo-region
-  - geo-location
-  - isolate-collection-date
-  - release-date
-  - update-date
-  - length
-  - host-name
-  - is-lab-host
-  - isolate-lineage-source
-  - bioprojects
-  - biosample-acc
-  - sra-accs
-  - submitter-names
-  - submitter-affiliation
+ppx_fetch:
+  seqs: https://lapis.pathoplexus.org/west-nile/sample/unalignedNucleotideSequences?versionStatus=LATEST_VERSION
+  meta: https://lapis.pathoplexus.org/west-nile/sample/details?dataFormat=csv&versionStatus=LATEST_VERSION
 
 # Config parameters related to the curate pipeline
 curate:
   # The path to the local geolocation rules within the pathogen repo
   # The path should be relative to the ingest directory.
   local_geolocation_rules: "defaults/geolocation-rules.tsv"
-  # The original field names should match the ncbi_datasets_fields provided above.
   # This is the first step in the pipeline, so any references to field names in the configs below should use the new field names
   field_map:
-    accession: accession
-    accession_version: accession_version
-    sourcedb: database
-    isolate-lineage: strain
-    geo-region: region
-    geo-location: location
-    isolate-collection-date: date
-    release-date: date_released
-    update-date: date_updated
-    length: length
-    host-name: host
-    is-lab-host: is_lab_host
-    isolate-lineage-source: sample_type
-    biosample-acc: biosample_accessions
-    sra-accs: sra_accessions
-    submitter-names: full_authors
-    submitter-affiliation: institution
-  # Standardized strain name regex
-  # Currently accepts any characters because we do not have a clear standard for strain names across pathogens
-  strain_regex: "^.+$"
-  # Back up strain name field to use if "strain" doesn"t match regex above
-  strain_backup_fields: ["accession"]
+    accessionVersion: PPX_accession
+    insdcAccessionFull: INSDC_accession
+    insdcRawReadsAccession: sra_accession
+    displayName: strain
+    geoLocCountry: country
+    geoLocAdmin1: division
```

Comment:
Similar to https://github.com/nextstrain/rsv/issues/99, `geoLocAdmin1` is problematic with varying granularity and format. Ideally they'd be fixed in Pathoplexus, but for now we may have to process it with a custom script. [Example](https://next.nextstrain.org/community/victorlin/nextstrain-test/WNV/all-lineages?c=division&f_division=Albany%20Co.,%20Ny,New%20York%20City,Ny,%20Albany,New%20York):

- `Albany Co., Ny`
- `Ny, Albany`
- `New York`
- `New York City`

I believe all of these are simply `New York` in the [current dataset](https://next.nextstrain.org/WNV/all-lineages/?c=division&f_division=New%20York).